source/strings.tex

```diff
@@ -2238,7 +2238,7 @@
     // \ref{string.ops}, string operations
     constexpr const charT* c_str() const noexcept;
     constexpr const charT* data() const noexcept;
-    constexpr charT* data() noexcept;
+    constexpr charT*       data() noexcept;
```

Comment:
If we feel we should add spaces here, could you add them before `charT*` instead?  This way, the spaces take up exactly the space that the now-missing "const" occupies.  Which is probably what we want to highlight.